src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImpl.java

```diff
@@ -50,9 +50,16 @@ public void performRequestDetailLineValidation(ConcurRequestExtractRequestDetail
         if (requestDetailFileLineIsCashAdvance(detailFileLine)) {
             boolean lineValidationPassed = true;
             lineValidationPassed =  requestedCashAdvanceHasNotBeenClonedInFile(detailFileLine, uniqueRequestIdsInFile);
+            LOG.info("requestedCashAdvanceHasNotBeenClonedInFile = " + lineValidationPassed);
+            
             lineValidationPassed &= requestedCashAdvanceHasNotBeenUsedInExpenseReport(detailFileLine);
+            LOG.info("requestedCashAdvanceHasNotBeenUsedInExpenseReport = " + lineValidationPassed);
+            
             lineValidationPassed &= requestedCashAdvanceIsNotBeingDuplicated(detailFileLine);
+            LOG.info("requestedCashAdvanceIsNotBeingDuplicated = " + lineValidationPassed);
+            
             lineValidationPassed &= requestedCashAdvanceAccountingInformationIsValid(detailFileLine);
+            LOG.info("requestedCashAdvanceAccountingInformationIsValid = " + lineValidationPassed);
```

Comment:
these log statements should include the function name

Reply:
Function name  added.